### DISCUSSION TITLE
manage multiple plex server

### DISCUSSION THREAD

**Discussion by thenamelessthing** [chadwpalm/PrerollPlus#8] [Ideas]
I have 2 plex servers, one on my linux server and another on an nvidia shield. It might be useful to be able to manage more than one plex server. What do you think?

**Comment by chadwpalm** [comment]
You're using Docker, right? Just spin up a second container and configure that one for your second Plex server. 

You'll just need to map a different external port and create a second directory on the host to map the config volume so the configs stay separate.
